### ISSUE TITLE
problem with test coverage importing markdown-transform package

### ISSUE THREAD

**Issue by j4m3sb0mb** [accordproject/markdown-transform#142]
**Describe the bug**
same issue as this https://github.com/accordproject/cicero/issues/401
in markdown-common

**Comment by jeromesimeon** [comment]
I have an initial PR for this in #161 but not sure how to test it. This is purely based on the corresponding fix that was done for accordproject/cicero#401

@j4m3sb0mb Would you have a simple test package or example I could use to test that the fix works?

**Comment by jeromesimeon** [comment]
Fix in #161 was tested with the following sample npm package:

[test.zip](https://github.com/accordproject/markdown-transform/files/4171048/test.zip)

Seems to work fine. Before the fix:
```
bash-3.2$ npm run test:cov 
npm run test:cov 

> testjest@1.0.0 test:cov /Users/jeromesimeon/tmp/test
> jest --timeOut=10000 --coverage

PASS lib/mylib.test.js
  acceptance
    ✓ translate (13ms)

[999D------------------------------------------------------------------------------------------------|----------|----------|----------|----------|-------------------|
File                                                                                            |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
------------------------------------------------------------------------------------------------|----------|----------|----------|----------|-------------------|
All files                                                                                       |    39.37 |    34.85 |    48.78 |     39.2 |                   |
 Users/jeromesimeon/tmp/test/lib                                                                |      100 |      100 |      100 |      100 |                   |
  mylib.js                                                                                      |      100 |      100 |      100 |      100 |                   |
 home/travis/build/accordproject/markdown-transform/packages/markdown-common/src                |    37.14 |    34.85 |     47.5 |    36.93 |                   |
  CommonMarkTransformer.js                                                                      |     56.2 |    51.35 |       65 |    56.39 |... 49,350,352,355 |
  Stack.js                                                                                      |    86.67 |    88.89 |    85.71 |    86.67 |             72,89 |
  ToMarkdownStringVisitor.js                                                                    |     1.08 |        0 |        0 |     1.08 |... 39,240,242,244 |
 home/travis/build/accordproject/markdown-transform/packages/markdown-common/src/externalModels |      100 |      100 |      100 |      100 |                   |
  CommonMarkModel.js                                                                            |      100 |      100 |      100 |      100 |                   |
------------------------------------------------------------------------------------------------|----------|----------|----------|----------|-------------------|
Test Suites: 1 passed, 1 total
Tests:       1 passed, 1 total
Snapshots:   1 passed, 1 total
Time:        1.236s
Ran all test suites.
```

After the fix:
```
bash-3.2$ npm run test:cov 
npm run test:cov 

> testjest@1.0.0 test:cov /Users/jeromesimeon/tmp/test
> jest --timeOut=10000 --coverage

PASS lib/mylib.test.js
  acceptance
    ✓ translate (14ms)

[999D----------|----------|----------|----------|----------|-------------------|
File      |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
----------|----------|----------|----------|----------|-------------------|
All files |      100 |      100 |      100 |      100 |                   |
 mylib.js |      100 |      100 |      100 |      100 |                   |
----------|----------|----------|----------|----------|-------------------|
Test Suites: 1 passed, 1 total
Tests:       1 passed, 1 total
Snapshots:   1 passed, 1 total
Time:        0.9s, estimated 1s
Ran all test suites.
```
